services/shotgrid_common/utils.py

```diff
@@ -984,6 +992,12 @@ def update_ay_entity_custom_attributes(
             ay_entity.tags = [tag["name"] for tag in attrib_value]
         elif ay_attrib == "status":
             logging.warning("Status update is not supported yet.")
-            ay_entity.status = attrib_value
+            # TODO: Implement status update
```

Comment:
any plans when this will be available?

Reply:
Hey @fabiaserra, thanks for reaching out. We haven't had a push from other clients on this yet, but I've gone ahead and logged the issue right here [https://github.com/ynput/ayon-shotgrid/issues/93](https://github.com/ynput/ayon-shotgrid/issues/93). Totally get that bidirectional sync would be super useful – we're all for it too! Right now, we're juggling our resources, and since it's just been you asking for this feature, we might need to hold off until it climbs up the priority list a bit.

Reply:
The question is not so much about adding the bi-directional but about having an option in the API to set statuses!! That's not specific to Shotgrid, just the core functionality of AYON

Reply:
As long as you can tell me how to set statuses on entities, I can do it myself